uma_vasp/receiving_vasp.py

```diff
@@ -170,11 +170,17 @@ def handle_pay_request_callback(self, user_id: str):
 
         request: PayRequest
         try:
-            request_data = (
-                flask_request.get_data(as_text=True)
-                if flask_request.method == "POST"
-                else json.dumps(flask_request.args)
-            )
+            if flask_request.method == "POST":
+                request_data = flask_request.get_data(as_text=True)
+            else:
+                payreq = {}
+                if "amount" in flask_request.args:
```

Comment:
I think we added a parsing function for this in some other sdks. Maybe we should add that for python too (follow-up is fine)